packages/design-system/src/components/Accordion/Accordion.tsx

```diff
@@ -31,7 +31,7 @@ const handleKeyDown = (e) => {
 };
 
 // eslint-disable-next-line react/prop-types
-const Accordion: FunctionComponent<AccordionProps> = ({ bordered, children, className }) => {
+export const Accordion: FunctionComponent<AccordionProps> = ({ bordered, children, className }) => {
```

Comment:
This and the change below were needed to be able to import the Accordion component into the typescript example